apps/opencs/model/prefs/state.cpp

```diff
@@ -356,6 +356,8 @@ void CSMPrefs::State::declare()
         QKeySequence(Qt::ControlModifier | (int)Qt::MiddleButton));
     declareModifier ("scene-speed-modifier", "Speed Modifier", Qt::Key_Shift);
     declareShortcut ("scene-delete", "Delete Instance", QKeySequence(Qt::Key_Delete));
+    declareShortcut ("scene-instance-groundlevel", "Instance to Ground Level", QKeySequence(Qt::Key_G));
+    declareShortcut ("scene-instance-drop", "Drop Instance", QKeySequence(Qt::Key_H));
```

Comment:
The wording is a bit strange. "Ground level" could be interpreted as the floor in a building, which is obviously not the case here. Let's call it what it is: Terrain. We are using this term consistently throughout the editor anyway.

I was about to comment that "Drop Instance" is inaccurate too. Should contain "selected" and it is rather instances than instance. But then I noticed that the same problem is present elsewhere. I guess we can go along with that for now and look at this issue again at a later time.